src/AppInstallerCLICore/Workflows/MSStoreInstallerHandler.cpp

```diff
@@ -133,6 +134,11 @@ namespace AppInstaller::CLI::Workflow
             installOptions.CompletedInstallToastNotificationMode(AppInstallationToastNotificationMode::NoToast);
         }
 
+        if (Manifest::ConvertToScopeEnum(context.Args.GetArg(Execution::Args::Type::InstallScope)) == Manifest::ScopeEnum::Machine)

```

Comment:
>if (Manifest::ConvertToScopeEnum(context.Args.GetArg(Execution::Args::Type::InstallScope)) == Manifest::ScopeEnum::Machine) [](http://example.com/codeflow?start=8&length=123)

Do we need to check if the InstallScope arg exists before calling this to prevent an exception from being thrown?

Reply:
If arg does not exist, the GetArg returns empty, no exception is thrown. I copied this snippet from existing usage in other places.